lms/static/js/fixtures/instructor_dashboard/batchenrollment.html

```diff
@@ -0,0 +1,76 @@
+<section id="membership" class="idash-section active-section" aria-labelledby="header-membership">
+    <h2 class="hd hd-2" id="header-membership">Membership</h2>
+
+    <fieldset class="batch-enrollment membership-section">
+        <legend id="heading-batch-enrollment" class="hd hd-3">Batch Enrollment</legend>
+        <label>
+            Enter email addresses and/or usernames separated by new lines or commas.
+            You will not receive notifications for messages that cannot be delivered, so be sure to double-check
+            spelling.
+            <textarea rows="6" name="student-ids" placeholder="Email Addresses/Usernames" spellcheck="false"></textarea>
+        </label>
+        <input id="is_course_white_label" value="False" type="hidden">
+        <div class="enroll-option">
+            <label for="course-modes-list-selector" class="has-hint">Select a course mode:
+                <select id="course-modes-list-selector" name="course_mode" class="course-modes-list-selector">
+                <option value="honor">Honor Code Certificate</option>
+                <option value="verified">Verified Certificate</option>
+                </select>
+
+                <div class="hint dropdown-hint auto-enroll-hint">
+                    <span class="hint-caret"></span>
+                    <p id="auto-enroll-tip">
+                        Learners will be automatically enrolled in the course mode that you select.
+                        <br><br>
+                        If you select <b>Unenroll</b> below, you do not need to select a course mode.
+                    </p>
+                </div>
+            </label>
+        </div>
+        <div class="enroll-option">
+            <label class="has-hint">
+                <input name="auto-enroll" id="auto-enroll" value="Auto-Enroll" checked="yes"
+                       aria-describedby="heading-batch-enrollment" type="checkbox">
+                <span class="label-text">Auto Enroll</span>
```

Comment:
Enroll learners automatically

(If we want to leave it as "Auto enroll", make sure "enroll" isn't capitalized)